src/renderer/components/welcome.tsx

```diff
@@ -92,10 +92,9 @@ export class Welcome extends React.Component<WelcomeProps, Partial<WelcomeState>
     const twoDaysAgo = Date.now() - 172800000;
     const toDeleteAll: Array<string> = [];
 
-    Object.keys(suggestions).forEach((key) => {
-      const item = suggestions[key];
+    suggestions.forEach((item) => {
```

Comment:
On Line 89, if `this.props.state.suggestion` doesn't exist, `suggestions` defaults to an empty object. I think we should change that to an array because Suggestions is an array type 🤔 .

Seems like this is a problem with the instantiation in the `renderSuggestions()` function as well.

Reply:
Sorry, some how forgot I ever opened this PR.  I was curious why Typescript wasn't complaining and it's because the codepaths assured the `|| {}` would never happened. It's definitely assigned and when it's reassigned it is always to an `[]`. I fixed them up regardless.